uyuni-tools/last_update_export.py

```diff
@@ -0,0 +1,42 @@
+#! /usr/bin/python3
+
```

Comment:
Could you please ask some documentation about this script as header? Including description, instructions and intended usage?

Reply:
Added script header